front-end-master/landing_page/index.css

```diff
@@ -111,21 +112,15 @@ a, a:hover{
 	z-index: -1;
 }*/
 .land-cover-img {
-  z-index: -1;
-  left: 50%;
-  position: absolute;
-  transform: translate(-50%, 0);
-  min-height: 100%;
-  min-width: 100%;
-}
-@media (max-width: 400px){
-	.land-cover-img{
-		padding: 0;
-		/position: absolute;
-		z-index: -1;
-		/width: 200%;
-		/left: 15%;
-	}
+      z-index: -1;
+      left: 50%;
+      position: absolute;
+      min-height: 100%;
```

Comment:
Good work using relative % instead of px

Reply:
Thanks